### ISSUE TITLE
Instructions unclear, inaccurate, and inefficient. Here's a revised version for anyone who's confused.

### ISSUE THREAD

**Issue by areyoufeelingitnowmrkrebs** [eylles/pywal16-libadwaita#10]
There are three specific things you need for this to work

1. [pywal16](https://github.com/eylles/pywal16)
2. [Kvantum](https://github.com/tsujan/Kvantum/tree/master/Kvantum)
3. [Gradience](https://github.com/GradienceTeam/Gradience)

I'm going to assume you can take it upon yourself to get software installed.

pywal16: generates a 16 color palette from the dominant colors of an image
Kvantum: allows you to install, configure, and apply themes for the Qt framework
Gradience: allows you to install, configure, and apply themes for the GTK framework

pywal works by reading system templates and user defined templates, then generating files containing color schemes from them in ~/.cache/wal.

This repo contains 3 important files

1. pywal.json
2. pywal.kvconfig
3. pywal.svg

These files are templates for pywal that generate themes for Kvantum and Gradience. Clone the repo, move the contents of the templates folder into ~/.config/wal/templates, and then run pywal. You should now have a corresponding file in ~/.cache/wal for each of the templates you imported.

The next step is to symlink the generated files into the appropriate directories. The README says to copy them over (it also gives the wrong location for the json file), but this would require manually copying the generated files every time a new palette is generated by pywal. By creating symlinks, you are allowing for "on the fly" changes which should update most things every time pywal is ran. You may need to log out/in.

```
# Move to Gradience directory
cd ~/.config/presets/user

# Create symlink
ln -s ~/.cache/wal/pywal.json 

# Move to Kvantum directory
cd ~/.config/Kvantum

# Create theme directory and cd into it
mkdir pywal
cd pywal

# Create symlinks
ln -s ~/.cache/wal/pywal.kvconfig
ln -s ~/.cache/wal/pywal.svg
```

Now just open Gradience and Kvantum to set the themes that pywal generated for you, which should update on the fly whenever pywal is ran (mostly).

**Comment by cadalen** [comment]
Thank you!

**Comment by areyoufeelingitnowmrkrebs** [comment]
> Thank you!

Happy to help!

**Comment by Shykid002** [comment]
Thank you bro you are a genius

**Comment by Shykid002** [comment]
sorry my bad please i've seen the generated theme in kvantum i've set it as default yet my gtk apps like thunar  didn't work i am currently using whitsur gtk theme. thanks

**Comment by areyoufeelingitnowmrkrebs** [comment]
That's because Kvantum is for Qt apps dude. You need to use the Gradience
theme for GTK applications.

**Comment by areyoufeelingitnowmrkrebs** [comment]
If you read the original instructions, it tells you what GTK theme you need to download for Gradience.

**Comment by areyoufeelingitnowmrkrebs** [comment]
It won't work without Gradience so I wouldn't even worry about the theme
until you can get that installed.

**Comment by astrysm** [comment]
Does gradience still work with it being discontinued?

**Comment by eylles** [comment]
@astrysm no, gradience is dead hence the repo got these 2 scripts https://github.com/eylles/pywal16-libadwaita/blob/master/scripts/build-gradience.py and https://github.com/eylles/pywal16-libadwaita/blob/master/scripts/gradience2css.py which can take a pywal theme to spit out a gradience theme and take the gradience theme to spit out the same-ish css as gradience did for gtk3 and gtk4.

i need to complete the https://github.com/eylles/pywal16-libadwaita/blob/master/scripts/build-kvantum.py script to completely sundown the templates and maybe move into consolidating the scripts into a small library, perhaps even publish it to pypi
